x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/rest/SecurityRestFilterTests.java

```diff
@@ -258,5 +280,55 @@ public void testSanitizeHeaders() throws Exception {
         }
     }
 
+    public void testProcessWithWorkflow() throws Exception {
+        final Workflow workflow = randomFrom(WorkflowResolver.allWorkflows());
+        if (randomBoolean()) {
+            restHandler = new TestBaseRestHandler(randomFrom(workflow.allowedRestHandlers()));
+        } else {
+            restHandler = Mockito.mock(RestHandler.class);
+            when(restHandler.getConcreteRestHandler()).thenReturn(new TestBaseRestHandler(randomFrom(workflow.allowedRestHandlers())));
```

Comment:
I think this does not work anymore since we agreed to not check the wrapped restHandler?

Reply:
Good catch. Removed it. 👍